reference/markdown/Format-comparison.md

```diff
@@ -0,0 +1,34 @@
+Although the original [Markdown DITA](./Markdown-syntax-reference.md) format and the [MDITA](./MDITA-syntax-reference.md) format for _LwDITA_ share some common syntax, there are several differences to consider when choosing which format to use.
+
+<!-- Re-use short descriptions from syntax topics -->
+
+- In 2015, the original _DITA-OT Markdown_ plug-in introduced a series of conventions to convert [Markdown] content to DITA, and vice-versa. This Markdown flavor was called _“Markdown DITA”_. The `markdown` format adds several complementary constructs to represent DITA content in Markdown, beyond those proposed for the [MDITA](./MDITA-syntax-reference.md) format in the [Lightweight DITA][LwDITA] specification drafts.
+
+- In 2017, the Markdown plug-in was superseded by the _LwDITA_ plug-in, which was bundled with DITA-OT 3.0, and added new formats for [Lightweight DITA][LwDITA]. The `mdita` format implements the subset of Markdown features proposed in the latest specification drafts, but differs in some ways from the original [Markdown DITA](./Markdown-syntax-reference.md) format.
+
+The following table provides an overview of differences between the `markdown` and `mdita` formats.
+
+| Features                 | Markdown DITA                   | MDITA (LwDITA)             |
+| ------------------------ | ------------------------------- | -------------------------- |
+| Format                   | `markdown`                      | `mdita`                    |
+| LwDITA                   | –                               | ✔                          |
+| First ¶                  | Body ¶                          | Short description          |
+| Subeadings               | Nested topics                   | Sections                   |
+| Topic IDs                | Attributes or title             | Generated from title       |
```

Comment:
I think we should not use just "attributes" because those are too easy to confuse with XML or HTML5 attributes. The Markdown extension is called "attribute lists".

Reply:
Good point: “attributes” alone is certainly ambiguous here.

It's complicated by the fact that Pandoc calls them [header_attributes](https://pandoc.org/MANUAL.html#extension-header_attributes) but PHP Markdown Extra uses [special attributes](https://michelf.ca/projects/php-markdown/extra/#spe-attr) — perhaps because they may be used for things other than headers.

The docs there put it this way, so I think we could use “special attribute block”.

> The id, multiple class names, and other custom attributes can be combined by putting them all into the same special attribute block: